tests/test_filter_registry.py

```diff
@@ -40,45 +36,42 @@
 }
 
 
-def test_filter_registry():
+def test_filter_registry(tmp_path):
```

Comment:
so this works because of the keyword `tmp_path`?

Reply:
`tmp_path` is a built-in pytest fixture.
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
cool!